aikau/src/main/resources/alfresco/dialogs/AlfDialog.js

```diff
@@ -62,7 +62,31 @@ define(["dojo/_base/declare",
         function(declare, Dialog, AlfCore, CoreWidgetProcessing, ResizeMixin, topics, _FocusMixin, lang, sniff, array,
                  domConstruct, domClass, domStyle, domGeom, html, aspect, on, when, $) {
    
-   return declare([Dialog, AlfCore, CoreWidgetProcessing, ResizeMixin, _FocusMixin], {
+    /**
+    * This is a customization of the default dijit/Dialog implementation to support
+    * AKU-916, preventing dialogs from scrolling the page when opened.
+    */
+   var CustomDialog = declare([Dialog], {
```

Comment:
I have to admit - I don't actually understand why we need to create a custom dialog module and then extend it. Give that we were originally just extending Dialog, why not just extend the `show` method directly? Why the indirection? It feels unnecessary.